src/components/Admin/ClientField.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { mapStateToProps } from '../../redux/actions';
+
+const ClientField = ({ usersList, client }) => {
+  let fields;
+  switch (client) {
+    case 'passengers':
+      try {
+        fields = usersList.users.map(({ passenger }) => {
+          const {
+            // eslint-disable-next-line camelcase
+            id, phone_number, first_name, last_name,
+          } = passenger;
+          return (
```

Comment:
The return statements inside switch cases may be turned into separate components

Reply:
Ok

Reply:
@ArtyomKosakyan any progress on this point?